src/wlapplication_options.cc

```diff
@@ -419,10 +431,51 @@ static std::map<KeyboardShortcut, KeyboardShortcutInfo> shortcuts_ = {
                                                          keysym(SDLK_t),
                                                          "editor_tools",
                                                          []() { return _("Tools"); })},
+   {KeyboardShortcut::kEditorChangeHeight,
+    KeyboardShortcutInfo({KeyboardShortcutInfo::Scope::kEditor},
+                         keysym(SDLK_h),
+                         "editor_change_height",
+                         []() { return _("Change Height Tool"); })},
+   {KeyboardShortcut::kEditorRandomHeight,
+    KeyboardShortcutInfo({KeyboardShortcutInfo::Scope::kEditor},
+                         keysym(SDLK_b),
```

Comment:
I think having some tools with a modifier and some without is inconsistent. Maybe make all tools use Shift.

Reply:
I didn't want to change existing ones, but I agree consistency is much preferred, and I practically already selected shift for this purpose. But:

 - Can History become plain 'H', just like the Tools menu is plain "T"?
 - How to resolve Players vs. Port spaces? Can Players remain plain P or make Port spaces shift+X like X for seafaring stats?
 - Can Info remain plain 'I'?

Reply:
+1 for <kbd>H</kbd> = History

I think all tools should now use <kbd>Shift</kbd>, so <kbd>Shift</kbd>+<kbd>I</kbd> for Info and <kbd>Shift</kbd>+<kbd>P</kbd> for players. 
<kbd>X</kbd> is soldier stats, seafaring stats are <kbd>E</kbd>. So if <kbd>E</kbd> is available use that for port spaces, otherwise well it doesn't matter that much since it's not 1:1 identical to any other hotkey.

Reply:
OK, thanks. Will change it accordingly.

(Sorry, I forgot to verify Seafaring hotkey before sending the comment, but I also corrected it in the meantime)